app/src/main/res/resource/values-de/strings.xml

```diff
@@ -112,12 +114,12 @@
     <string name="feedback_exposure">Belichtungszeit</string>
     <string name="feedback_aperture">Blende</string>
     <string name="feedback_focal">Brennweite</string>
-    <string name="feedback_iso">Iso</string>
+    <string name="feedback_iso">ISO</string>
 
     <string name="feedback_portfolio_is_null">Portfolio Link ist ungültig.</string>
```

Comment:
Durchkoppeln? Okay, no, I'd rather write it as one word: "Portfoliolink". Or, when you don't like that, "Link zum Portfolio".